docs/reference/cloud_event_list.md

```diff
@@ -0,0 +1,44 @@
+
+Unless otherwise noted, the `source` field of the CloudEvent is the public key of the host from which the event originated.
+
+| Type | Fields | Description |
+| :-- | --- | :-- |
+| `actor_start_failed` | `actor_ref`<br/>`error` | Indicates a failed attempt to start an actor |
+| `actor_stopped` | `public_key`<br/>`instance_id`<br/>`annotations` | Indicates a single actor instance stopped on a host |
+| `actor_started` | `public_key`<br />`image_ref`<br/>`api_version`<br/>`instance_id`<br/>`annotations`(map)<br/>`claims`(map)| A single actor instance started |
+| `actor_updated` | `public_key`<br/>`revision`<br/>`instance_id`<br/> | An actor "live update" operation completed successfully |
+| `actor_update_failed` | `public_key`<br/>`revision`<br/>`instance_id`<br/>`reason` | A live update operation failed |
+| `provider_started` | `public_key`<br/>`image_ref`<br/>`link_name`<br/>`contract_id`<br/>`instance_id`<br/>`annotations`<br/>`claims`(map) | A capability provider succesfully started |
+| `provider_start_failed` | `provider_ref`<br/>`link_name`<br/>`error` | A capability provider failed to start |
+| `provider_stopped` | `public_key`<br/>`link_name`<br/>`contract_id`<br/>`instance_id`<br/>`annotations`<br/>`reason` | A capability provider stopped |
+| `host_started` | `labels`<br/>`friendly_name`<br/>`uptime_seconds`<br/>`version` | Announced by a host after completing initialization |
+| `host_stopped` | `labels` | Announced by a host just prior to final termination |
+| `host_heartbeat` | `actors`<br/>`providers`<br/>`labels`<br/>`friendly_name`<br/>`version`<br/>`uptime_seconds`<br/>`uptime_human` | Periodically emitted by hosts to advertise inventory and status |
+| `health_check_passed` | `public_key`<br/>`contract_id`<br/>`link_name` | A provider health check passed |
+| `health_check_failed` | `public_key`<br/>`contract_id`<br/>`link_name` | A provider health check failed |
+| `health_check_status` | `public_key`<br/>`contract_id`<br/>`link_name` | Provider health status unchanged |
+| `refmap_set` | `oci_url`<br/>`public_key` | Event advertising a reference mapping being cached |
+| `linkdef_set` | `id`<br/>`actor_id`<br/>`provider_id`<br/>`link_name`<br/>`contract_id`<br/>`values`(map) | Event advertising a link definition being cached |
+| `linkdef_deleted` | `id`<br/>`actor_id`<br/>`provider_id`<br/>`link_name`<br/>`contract_id`<br/>`values`(map) | Event advertising link definition being deleted from cache |
+| `actors_started` | `public_key`<br/>`image_ref`<br/>`annotations`<br/>`host_id`<br/>`count`<br/>`claims`(map) | A batch of actor instances successfully started. `host_id` is the same as the envelope's `source` |
+| `actors_start_failed` | `public_key`<br/>`image_ref`<br/>`annotations`<br/>`host_id`<br/>`error` | A batch of actor instances failed to start. `host_id` is the same as the envelope's `source` |
```

Comment:
nit: thoughts on moving these up to be next to the other actors events? actors started. start failed, and stopped?

Reply:
done